cypress/integration/zDestructive.spec.js

```diff
@@ -0,0 +1,68 @@
+import { visitOneAndOnlyCluster, testInfraClusterName } from './shared';
+
+// Theese tests changes state of the cluster permanently.
+// If enabled, they are called at last.
+describe('Destructive tests need to be enabled manually', () => {
+  // TODO: add more thorough checks for field-validations
+  it('Cluster can be installed', () => {
```

Comment:
Considering recent state of the project, this is not a destructive test yet. So we can keep it enabled for now. The idea is to switch to `xit` later.